### PR TITLE
2020 07 08 add jenkins deploy stage

### PR DESCRIPTION
**Pull request by yalturmes**
* Updated Jenkinsfile vars
* Added deployment stage for develop and master branches.
* develop branch scenario deployment to dev environment was tested successfully:  https://jenkins.rdpc-dev.cancercollaboratory.org/blue/organizations/jenkins/provision%2Fhelm/detail/helm/66/pipeline

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -66,20 +85,36 @@ spec:
                 container('docker') {
                     withCredentials([usernamePassword(credentialsId: 'argoGithub', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
                         sh "git tag ${version}"
-                      sh "git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/icgc-argo/workflow-gateway --tags"
+                      sh "git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/${githubRepo} --tags"
```
Comment (andricDu) [inline Jenkinsfile]: could you make `githubRepo` -> `gitHubRepo` just to follow our camel case convention.

I've made this change elsewhere as I am normalizing our params.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,3 +1,6 @@
+def dockerHubRepo = "icgcargo/workflow-gateway"
+def githubRepo = "icgc-argo/workflow-gateway"
```
Comment (andricDu) [inline Jenkinsfile]: You'll need to change it here too.

**Review by andricDu — CHANGES_REQUESTED** [review]

**Review by lepsalex — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (yalturmes) [reply Jenkinsfile]: Updated. Will stick with the camel-case moving forward.

File: Jenkinsfile (hunk 2 above)
Comment (yalturmes) [reply Jenkinsfile]: Updated.

**Review by andricDu — APPROVED** [review]

**Pull request merged**